src/modules/endpoint/reducers/citiesReducer.js

```diff
@@ -8,6 +8,10 @@ export default (
   state: CitiesStateType = defaultCitiesState, action: CitiesActionType
 ): CitiesStateType => {
   switch (action.type) {
+    case 'FETCH_CITIES':
+      return {
+        models: null
```

Comment:
I have a discussion here: So if we discard data right when we start loading then it will be lost in case of an error.
I think this is oke but it would be better if the previous data is shown in that case. What do you think?

Reply:
Well, it should not be lost completely, since the datacontainer / database might still hold the old data... But yeah, I see your problem, that we might want to display data when a refresh failed (if there's offline data available). So we might refill the state with the old data if available (and for example notify the user of the error by using a Toast triggered in a Saga?)
Of course this is not part of this task, although we definitely need to discuss on it...

Reply:
I created a task for this: https://issues.integreat-app.de/browse/NATIVE-334